site/js/modules/webui/render.js

```diff
@@ -243,6 +243,37 @@ function submitInputs(params, context, container, inputs, onSubmitFunc) {
     ;
 }
 
+// Set the page title given a list of title parts.
+// Each page title part is [display name, optional link].
+// If title parts is empty, the page title defaults to the tab title.
+function makeTitle(tabTitle, pageTitleParts = []) {
+    let titlePartsHTML = [];
+    for (const part of pageTitleParts) {
+        let displayName = part[0];
+        let link = part[1];
+
+        if (link) {
+            titlePartsHTML.push(`<a href='${link}'>${displayName}</a>`);
+        } else {
+            titlePartsHTML.push(`${displayName}`);
```

Comment:
Why the string wrap at all?

`titlePartsHTML.push(displayName);`
vs
``titlePartsHTML.push(`${displayName}`);``